sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -255,7 +258,17 @@ protected AmqpConnectionScope()
             var connection = await ActiveConnection.GetOrCreateAsync(timeout).ConfigureAwait(false);
             cancellationToken.ThrowIfCancellationRequested<TaskCanceledException>();
 
-            var link = await CreateReceivingLinkAsync(connection, consumerEndpoint, eventPosition, consumerOptions, timeout.CalculateRemaining(stopWatch.Elapsed), cancellationToken).ConfigureAwait(false);
+            var link = await CreateReceivingLinkAsync(
+                connection,
+                consumerEndpoint,
+                eventPosition,
+                timeout.CalculateRemaining(stopWatch.Elapsed),
+                (uint)prefetchCount,
```

Comment:
Good question.  That was the pattern used by Track One, we had just kind of followed along.   Since we're not currently exposing to developers, I converted them all to `unit`.   When we introduce the `PartitionReceiverClient` we can consider what to expose for the options type.